x-pack/platform/plugins/shared/maps/ui_tests/tests/full_screen_mode.spec.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, tags, test, ANNOTATIONS } from '@kbn/scout';
+import {
+  VISIBLE_CHROME,
+  HIDDEN_CHROME,
+  FULL_SCREEN_MODE,
+  EXIT_FULL_SCREEN,
+} from '../fixtures/constants';
+
+test.describe(
+  'Maps',
+  {
+    tag: tags.DEPLOYMENT_AGNOSTIC,
+    annotation: { type: ANNOTATIONS.WARNING, description: 'Contains deprecated goto().' },
```

Comment:
could you share more about this line?

I'm not sure we should add the logic we don't handle on report side. cc @dolaru knows more about reporter plans

Reply:
The Scout events reporter is capturing tags and annotations. 

If you want to formalize the annotation types for consistency, I'd rather have them defined in the `kbn-scout-info` package to avoid circular dependencies.

Tags should be used for categorization/grouping and annotations should be used to add information.